Framework/PythonInterface/plugins/algorithms/ReorientUnitCell.py

```
@@ -211,16 +229,28 @@ def PyExec(self):
+            try:
+                # Try to recalculate error, but if it fails (e.g. due to too few peaks) we will still have the reoriented cell
+                (
+                    TransformHKL(
+                        PeaksWorkspace=wsname,
+                        Tolerance=self.getProperty("Tolerance").value,
+                        HKLTransform=optimal_transform,
+                        FindError=True,
+                    ),
+                )
+                self.log().notice("Successfully recalculated error after reorienting unit cell.")
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)